app/src/main/java/com/example/naottae/ui/todayReserv/dialog/ChangeCancelResonDialog.kt

```
@@ -0,0 +1,86 @@
+    ): View? {
+        val view = inflater.inflate(R.layout.dialog_cancel_reason, container, false)
+
+        val etReason = view.findViewById<EditText>(R.id.etReason)
+        val btnSubmit = view.findViewById<Button>(R.id.btnSubmit)
+        val tvCharCount = view.findViewById<TextView>(R.id.tvCharCount)
+        val btnClose = view.findViewById<ImageView>(R.id.btnClose)
+
+        btnClose.setOnClickListener { dismiss() }
+
+        etReason.addTextChangedListener(object : TextWatcher {
+            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)